### PR TITLE
Patch GH automation

### PR DESCRIPTION
**Pull request by davidmrdavid**
This PR fixes our GH automation so that new issues have the "needs: triage" label added automatically again. This stopped working recently when FabricBot migrated to a new config language

### CODE REVIEW

**Review by nytian — APPROVED** [review]

**Comment by davidmrdavid** [comment]
@cgillum: seems we need an approval w/ someone with write access

**Review by cgillum — APPROVED** [review]
I re-ran the CI and it's green now. Will merge.

**Pull request merged**
